services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/subscription/SubscriptionManagerImpl.java

```diff
@@ -67,7 +67,16 @@ public SubscriptionPagedResults getList(UserInfo userInfo, SubscriptionRequest r
 		ValidateArgument.required(request, "request");
 		ValidateArgument.required(request.getObjectType(), "SubscriptionRequest.objectType");
 		ValidateArgument.required(request.getIdList(), "SubscriptionRequest.idList");
-		return subscriptionDao.getSubscriptionList(userInfo.getId().toString(), request.getObjectType(), request.getIdList());
+		switch(request.getObjectType()) {
+		case THREAD:
+			return subscriptionDao.listSubscriptionForThread(userInfo.getId().toString(), request.getIdList());
+		case FORUM:
+			return subscriptionDao.listSubscriptionForForum(userInfo.getId().toString(), request.getIdList());
+		case DATA_ACCESS_SUBMISSION_STATUS:
```

Comment:
I thought thread and forum where the special cases and all others could use the generic list subscriptions.  Should the default call the generic list subscriptions?